In [1]:
import pandas as pd
import numpy as np
import scipy.stats as stats

In [2]:
data=pd.read_csv('/content/drive/MyDrive/Sales_add.csv')


In [3]:
data

Month      Region      Manager  Sales_before_digital_add(in $)  \
0    Month-1  Region - A  Manager - A                          132921   
1    Month-2  Region - A  Manager - C                          149559   
2    Month-3  Region - B  Manager - A                          146278   
3    Month-4  Region - B  Manager - B                          152167   
4    Month-5  Region - C  Manager - B                          159525   
5    Month-6  Region - A  Manager - B                          137163   
6    Month-7  Region - C  Manager - C                          130625   
7    Month-8  Region - A  Manager - A                          131140   
8    Month-9  Region - B  Manager - C                          171259   
9   Month-10  Region - C  Manager - B                          141956   
10  Month-11  Region - B  Manager - C                          159339   
11  Month-12  Region - A  Manager - A                          178939   
12  Month-13  Region - B  Manager - A                          145062   
13  Month-14  Region - A  Manager - A                          151514   
14  Month-15  Region - C  Manager - A                          147463   
15  Month-16  Region - A  Manager - B                          177195   
16  Month-17  Region - A  Manager - C                          140862   
17  Month-18  Region - C  Manager - B                          167996   
18  Month-19  Region - B  Manager - A                          132135   
19  Month-20  Region - A  Manager - B                          152493   
20  Month-21  Region - B  Manager - A                          147425   
21  Month-22  Region - A  Manager - C                          130263   

    Sales_After_digital_add(in $)  
0                          270390  
1                          223334  
2                          244243  
3                          231808  
4                          258402  
5                          256948  
6                          222106  
7                          230637  
8                          226261  
9                          193735  
10                         203175  
11                         276279  
12                         205180  
13                         253131  
14                         229336  
15                         187305  
16                         234908  
17                         191517  
18                         227040  
19                         212579  
20                         263388  
21                         243020

#**The company wishes to clarify whether there is any increase in sales after stepping into digital marketing**



H0 :There is no increase in sales after stepping into digital marketing



H1 :There is an increase in sales after stepping into digital marketing

Level of significant as alpha = 0.05


In [9]:
t_value, p_value = stats.ttest_ind(data['Sales_After_digital_add(in $)'], data['Sales_before_digital_add(in $)'], alternative='greater')

In [10]:
print("The t value is:", t_value)
print("The p value is: ", p_value)

The t value is: 12.995084451110877
The p value is:  1.3071840034523225e-16





*   The p value is lesser than 0.05.
*   The alternative hypothesis is true, so reject null hypothesis
 






# The company needs to check whether there is any dependency between the features “Region” and “Manager”

 H0: There is no dependency between Region and Manager

 Ha: There is dependency between Region and Manager

In [15]:
crosstab = pd.crosstab(data["Region"],data["Manager"])
crosstab


Manager     Manager - A  Manager - B  Manager - C
Region                                           
Region - A            4            3            3
Region - B            4            1            2
Region - C            1            3            1

In [17]:
stats.chi2_contingency(crosstab)

(3.050566893424036,
 0.5493991051158094,
 4,
 array([[4.09090909, 3.18181818, 2.72727273],
        [2.86363636, 2.22727273, 1.90909091],
        [2.04545455, 1.59090909, 1.36363636]]))

p_value=0.5493991051158094



   

*    p_value is greater than 0.05.

*    we cannot reject the null hypothesis. 

*   There is no dependency between the features "Region" and "Manager"

